### ISSUE TITLE
Machine generated glosses

### ISSUE THREAD

**Issue by arrocke** [arrocke/gloss-translation#64]

**Comment by arrocke** [comment]
@BethedenMinistries What research needs to be done before we can plan this out?

**Comment by BethedenMinistries** [comment]
Are we talking about machine translation from a base interlinear, such as English?  If so, we need to research how AI is changing this space and perhaps enabling it to be more nuanced. As far as I can tell, the current interlinear translates each gloss out of context, which severely limits its accuracy. It would be good to look into the possibility of having AI (like the GPT-4 API) read the context of the verse in a literal translation, read the gloss in context, and then make its guess based on that.  
The BibleHub interlinear in English will be coming out at the end of this year, and it will be open and vastly better than the current one. So we could use what we have for now, after @LouKehler does some cleanup on it, and then use the new BibleHub one when it's released (if it's better than the one we clean up). 
If we use the Spanish as a base, we need to run tests comparing how well it does compared to using English as a base. We'd need to test it being translated to Western languages and then Eastern languages.

**Comment by arrocke** [comment]
closing in favor of #153
